build/devenv/tests/e2e/smoke_test.go

```diff
@@ -703,3 +691,118 @@ func multiVerifierTestCases(t *testing.T, src, dest uint64, in *ccv.Cfg, c *evm.
 		},
 	}
 }
+
+// ============================================================================
+// RMN Curse Helper Functions
+// ============================================================================
+
+// chainSelectorToSubject converts a chain selector to a bytes16 curse subject.
+func chainSelectorToSubject(chainSel uint64) [16]byte {
+	var result [16]byte
+	// Convert the uint64 to bytes and place it in the last 8 bytes of the array
+	binary.BigEndian.PutUint64(result[8:], chainSel)
+	return result
+}
+
+// globalCurseSubject returns the global curse constant.
+func globalCurseSubject() [16]byte {
+	return globals.GlobalCurseSubject()
+}
```

Comment:
If I recall these are also defined somewhere in the verifier package - maybe export and share?